R/cop_pse_auxiliary.R

```diff
@@ -66,7 +66,7 @@ cal_delta_star <- function(FR2max,
                            est_uncond, 
                            rxz, 
                            n_obs) {
-    if (FR2max > .99) {FR2max <- .99}
+    if (FR2max > .99) {FR2max =.99}
```

Comment:
did this possibly break the tests/R CMD check?

Reply:
It is what I wanna talk with you ,when I change the = to "<-", there is a tip to remind me this line can't go through the  test.

Reply:
but when you see [e16bc75], I changed the "=" back to "<-",still show the same issue. So I don't think the line 69 break the tests.